packages/account/src/Sections/Security/Passkeys/passkeys-configs.tsx

```diff
@@ -117,6 +117,13 @@ type TGetModalContent = {
 };
 
 export const getModalContent = ({ error, is_passkey_registration_started }: TGetModalContent) => {
+    const error_message =
+        (error as TServerError)?.name === 'NotSupportedError' ? (
```

Comment:
**NIT**: Can we move error codes to `packages/account/src/Constants/api-error-codes.ts`  so it will be easy to track?